### ISSUE TITLE
pull:events Import Returns an Error and Breaks Events, Could Use Some Proactive Error Checking / Validation

### ISSUE THREAD

**Issue by allella** [hackgvl/hackgreenville-com#113]
@zach2825 as with #109 it seemed the calendar and events lists were building up some cancellation crud. I think the root problem may be related to the events imports breaking.

The _orgs_ import still works, but I tried to clear and reimport and now we only show events for early May.

I ran a couple commands from back with our https://github.com/codeforgreenville/hackgreenville-com/issues/99 conversation to get to this current situation.
```
php artisan migrate:fresh --seed
php artisan --fix pull:events; php artisan pull:orgs;
```

The error I see on the production command line when running any version of pull:events is
![image](https://user-images.githubusercontent.com/1777776/166707233-5016cd63-0581-4890-8bf0-d842eab73e69.png)

**Comment by allella** [comment]
I'm going to leave this issue open, mainly so we can add some error checking, but I've confirmed the source is missing service and service_id values in the events API.

**Comment by allella** [comment]
I've patched this issue and it fixed the two in-person Eventbrite events which were missing those _service_ and _service_id_ values.

I think I'm going to change the import cronjob to email me any errors so these things don't go unnoticed. Though, if we added some proactive error checking I'm curious if there's a lightweight way to email an admin when the PHP code catches a breaking change.

The new [What's Brewing Greenville event is now showing](https://hackgreenville.com/calendar).
